apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx

```diff
@@ -158,7 +160,12 @@ function AssetCentricGraphCard({
                       minHeight={25}
                     >
                       {items[1].value !== undefined ? (
-                        <CurrencyUnitValue {...items[1]} />
+                        <CurrencyUnitValue
```

Comment:
> infering types from variable is kinda dangerous as you can change the items array to any type here

Infering is not that dangerous, ts will reject non-overlapping types when trying to cast.

> Also I think it's better to …

Or remove the casting since it works without 😉